packages/trpc/server/routers/viewer/workflows/update.handler.ts

```diff
@@ -92,6 +92,7 @@ export const updateHandler = async ({ ctx, input }: UpdateOptions) => {
       id: {
         in: activeOn,
       },
+      parentId: null,
```

Comment:
We are adding the children later in `activeOnWithChildren`. On the first update when the managed event type is added only the parentId is coming in